hooks.lua

```diff
@@ -49,7 +66,7 @@ function OnUpdatingSign(World, BlockX, BlockY, BlockZ, Line1, Line2, Line3, Line
 		if (not(Player:HasPermission("es.warpsign") == true)) then
 			return true
 		elseif (Line2 == "") then
-			Player:SendMessageFailure('Must supply a tag for the warp.')
+			Player:SendMessageFailure('Must supply a tag for taadfs	he warp.')
```

Comment:
thanks @NiLSPACE I think the wrong spelled the comes from editing on my mobile phone. the spaces from copy paste, I will correct both